src/V3AstNodes.h

```diff
@@ -2436,6 +2467,15 @@ class AstModule : public AstNodeModule {
     virtual string verilogKwd() const { return "module"; }
 };
 
+class AstProgram : public AstNodeModule {
+    // A program declaration
```

Comment:
This is a fine change, I suggest making a separate pull request to add and use AstProgram, as is an easy thing to get out of the way.